python/tests/kat/t_tracing.py

```diff
@@ -680,127 +680,3 @@ def check(self):
         traceId = trace['traceId']
         assert len(traceId) == 32
 
-class TracingTestZipkinV1(AmbassadorTest):
```

Comment:
Like the `protocol_version` tests, it would probably be good for this test to stay, but to have it check that the config is rejected.